src/app/View/Components/FormModal.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Backpack\CRUD\app\View\Components;
+
+class FormModal extends Form
+{
+    /**
+     * Create a new component instance.
+     *
+     * @param  string  $controller  The CRUD controller class name
+     * @param  string  $operation  The operation to use (create, update, etc.)
+     * @param  string|null  $action  Custom form action URL
+     * @param  string  $method  Form method (post, put, etc.)
+     * @param  string  $modalTitle  Title for the modal
+     * @param  string  $modalClasses  CSS classes for the modal dialog
+     * @param  string  $formRouteOperation  The operation to use for the form route (defaults to 'create')
+     * @param  string  $id  The ID for the form element (defaults to 'backpack-form')
+     */
+    public function __construct(
+        public string $controller,
+        public string $id = 'backpack-form',
+        public string $operation = 'create',
+        public string $formRouteOperation = 'create',
+        public ?string $action = null,
+        public string $method = 'post',
+        public string $modalTitle = 'Form',
+        public string $modalClasses = 'modal-dialog modal-lg',
```

Comment:
Any reason to have `modal` in the names here, and not:

```suggestion
        public string $title = 'Form',
        public string $classes = 'modal-dialog modal-lg',
```

I mean the component is already called `form-modal` so when you pass the `title` and `classes` I think it's intuitive that it will be the title and classes of that modal.